VirtoCommerce.Platform.Web/Scripts/app/dynamicProperties/dynamicProperties.js

```diff
@@ -56,4 +56,13 @@
 
         return retVal;
     }
-});
\ No newline at end of file
+})
```

Comment:
I don't understand why we need this code, we need the display Description for a dynamic property as well as DisplayName, I don see any differences, why we required this code then?

Reply:
This filter is strange. He has a name that speak about a localization of name but factually besides localization he build a formatted string. We need a various logic of view for description and name (in this case name has parentheses and description are not). At logic in one filter we will have a hard-code for description and name and other properties. A new filter don't do this. And old filter will work on.